src/parallel.rs

```diff
@@ -4,14 +4,12 @@ cfg_if! {
     if #[cfg(feature = "rayon")] {
         use rayon::slice;
         pub use rayon::prelude::*;
-
```

Comment:
any reason for removing the whitespace here?

Reply:
I guess this happened as I was trying to add another optional rayon feature. I added it again.